.travis.yml

```diff
@@ -5,7 +5,6 @@ cache:
         - $HOME/.composer/cache
 
 php:
-  - 5.3
   - 5.4
```

Comment:
Maybe we could also drop the support of php5.4 as it's also not supported anymore ?